target/linux/generic/backport-5.15/703-10-v5.16-net-dsa-introduce-helpers-for-iterating-through-port.patch

```diff
@@ -31,9 +31,9 @@ Signed-off-by: David S. Miller <davem@davemloft.net>
 
 --- a/include/net/dsa.h
 +++ b/include/net/dsa.h
-@@ -476,6 +476,34 @@ static inline bool dsa_is_user_port(stru
- 	return dsa_to_port(ds, p)->type == DSA_PORT_TYPE_USER;
- }
+@@ -504,6 +504,34 @@ static inline bool dsa_is_user_port(stru
+ 	dsa_switch_for_each_port((_dp), (_ds)) \
+ 		if (dsa_port_is_cpu((_dp)))
```

Comment:
This patch is not needed any more. It was upstreamed here: https://github.com/gregkh/linux/commit/9902f91cf666124a6b50bbcf483b46ecb09ef408

Reply:
I removed it, thank you.  I did not re-do the update since 5.15.116 is out now... doing it there.  Please let me know if that is unacceptable.